packages/core/src/util/postHog.ts

```diff
@@ -0,0 +1,16 @@
+import { PostHog } from "posthog-node";
+
+const POSTHOG_API_KEY = "{POSTHOG_API_KEY}";
```

Comment:
@psugihara here is where we would inject the string during the build.  do I want to be looking at an swc config to do that?  I saw that swc has a transform option that sounds like maybe what i want? https://swc.rs/docs/configuration/swcrc

Reply:
I think you actually want to do something with rollup/preconstruct.

swc is just used 1. internally by next.js 2. to parse templates at runtime in the CLI